client/server/sanitize/index.js

```diff
@@ -47,7 +47,7 @@ function jsSlashEncoder( charStr ) {
  * @returns {string} JSON serialized string
  **/
 exports.jsonStringifyForHtml = function ( value ) {
-	const jsonInHtmlFilter = /[^\x22,\-.0-9:A-Z[\x5C]_a-z{}]/g;
+	const jsonInHtmlFilter = /[^\x22,\-.0-9:A-Z[\x5C\]_a-z{}]/g;
```

Comment:
The `[` following `A-Z` might also need escaping.

Reply:
I think this is fine, it's usually already escaped (in the same way `.` doesn't need to be escaped). But I'll add a backslash for good measure.

Reply:
It looks like there are lint errors when trying to add backslashes for those characters. That's likely why this bug was introduced :)
```
  50:37  error  Unnecessary escape character: \.  no-useless-escape
  50:46  error  Unnecessary escape character: \[  no-useless-escape
```